src/frontend/admin/src/layouts/dashboard/nav/DashboardNav.tsx

```diff
@@ -52,6 +34,29 @@ export function DashboardNav(props: Props) {
         <DashboardNavHeader />
         <DashboardNavContent onChangeRoute={props.handleClose} />
       </Drawer>
-    </>
+    );
+  }
+
+  return (
```

Comment:
I don't really understand how these changes make the overlay disappear? I may get something wrong.